lib/postgresql-store.js

```diff
@@ -401,8 +401,14 @@ module.exports = function (opts) {
     var wherestr = ''
 
     if (!_.isEmpty(w) && w.params.length > 0) {
-      w.params.forEach(function (param) {
-        params.push('"' + escapeStr(camelToSnakeCase(param)) + '"=$' + (++cnt))
+      w.params.forEach(function (param, i) {
+        if(w.values[i] === null || w.values[i] === '') {
```

Comment:
@bmullan91 empty string should not create a WHERE IS NULL criteria.

empty string is a valid value and we reduce the functionality if we do that.